protocol/v2/qbft/instance/commit.go

```diff
@@ -45,7 +45,7 @@ func (i *Instance) UponCommit(signedCommit *specqbft.SignedMessage, commitMsgCon
 			return false, nil, nil, errors.Wrap(err, "could not get msg commit data")
 		}
 
-		agg, err := aggregateCommitMsgs(commitMsgs)
+		agg, err := AggregateCommitMsgs(commitMsgs)
```

Comment:
@lior-blox this is a diff from spec FYI.

Reply:
added a note

Reply:
@alonmuroch Maybe it should be moved to signedMessage in spec, as its not really an commit function and more of signedMessage function?